Sprint-3/revise/implement/card-validator.test.js

```diff
@@ -0,0 +1,23 @@
+
+const cardValidator = require("./cardValidator");
+
+
+test("Checks if the number has 16 digits.", () => {
+    expect(cardValidator("1243578569873476")).toEqual(true)
```

Comment:
The function can successfully check if there are 16 digits in the card number does not mean that the function can return false when the card number has 15 or 20 digits.